src/pass.h

```diff
@@ -104,22 +118,22 @@ class Pass : public WasmWalker {
 // e.g. through PassRunner::getLast
 
 // Handles names in a module, in particular adding names without duplicates
-class NameManager : public Pass {
+class NameManager : public WalkerPass<WasmWalker<NameManager, void> > {
```

Comment:
Didn't realize C++ got so advanced over the years. If I run clang-format, it will change quite a bit of the file. Perhaps I can submit another PR with a style change, @kripken what do you think?